fennel/test_lib/mock_client.py

```diff
@@ -524,6 +541,28 @@ def extract_features(
             extractors, input_dataframe, output_feature_list, timestamps
         )
 
+    def extract_historical(
+        self,
+        inputs: List[Union[Feature, Featureset, str]],
```

Comment:
I thought inputs was only supposed to be list of features, not featuresets? cc @aditya-nambiar

Reply:
Yes thats right. We have a check for that in this function itself, but makes sense to remove it from the type system since this code also serves as documentation. @nonibansal

Reply:
Removed.